mtp_noms_ops/assets-src/stylesheets/views/_security-form-fields.scss

```diff
@@ -93,3 +93,35 @@ select[disabled] {
     }
   }
 }
+
+#filter-senders {
+  .column-one-third {
+    .mtp-info-box {
+      padding: 1em;
+
+      .mtp-info-box__heading {
+        font-size: 19px;
+      }
+
+      p {
+        margin-top: em(20, 16);
+        margin-bottom: 0;
+        font-size: 16px;
+      }
+    }
+  }
+}
+
+.mtp-security-form--simple {
+  #id_search-label {
+    font-size: 27px;
```

Comment:
just an fyi: gds fonts shrink on smaller screens. this label will remain large because of the absolute size. in this case, i'd leave it as you have done. it still looks perfectly fine on mobile